conversion/conversion.go

```diff
@@ -587,6 +611,25 @@ func WriteSessionFile(conv *internal.Conv, name string, out *os.File) {
 	fmt.Fprintf(out, "Wrote session to file '%s'.\n", name)
 }
 
+// WriteConvGeneratedFiles creates a directory labeled downloads with the current timestamp
+// where it writes the sessionfile, report summary and DDLs then returns the directory where it writes.
+func WriteConvGeneratedFiles(conv *internal.Conv, dbName string, driver string, BytesRead int64, out *os.File) (string, error) {
+	now := time.Now()
+	dirPath := "harbour_bridge_downloads/" + dbName + "/"
```

Comment:
how about we call this directory "output"? since these aren't necessary downloaded files but rather output from running the command